config.yml

```
@@ -154,6 +154,8 @@ donation:
       details: "When creating a character, you get a choice of hats in the creation menu which you spawn with every time until your donorship expires."
     - caption: "Quiet Round Mode"
       details: "When creating a character, you get the choice of toggling the quiet round mode. You will opt out of becoming an antag, but also never be an antags target. It's a round perfect for those wishing to learn a job role peacefully. Of course, we cannot guarantee you will not be collateral!"
+    - caption: "Donor Chat"
+      details: "You get to use a private chat only donors and staff can see. In chat, type \"donor\" then spacebar to toggle donorchat (getting rid of say\")"

```

```suggestion
      details: "You get to use a private chat only donors and staff can see. In chat, type \"donor\" then spacebar to toggle donorchat (getting rid of \"say\")"

```

I think this was a typo in the original too?

---

The single quote is intentional, say is the command, the open quote is the start of the argument